chips/stm32f4xx/src/clocks/clocks.rs

```diff
@@ -515,6 +536,30 @@ impl<'a, ChipSpecs: ChipSpecsTrait> Clocks<'a, ChipSpecs> {
     }
 }
 
+/// Clocks controller trait
+///
+/// This can be used to control clocks without the need to keep a reference of the chip specific
+/// Clocks struct, for instance by peripherals
+pub trait ClocksCtrl {
```

Comment:
I went for `ClocksCtrl` since it may be extended with writer methods in future. However I see now that it can be confusing given that the current main intent of that trait is to only retrieve information from the clocks module. So I renamed it to `Stm32f4Clocks`.